optimizer_wrapper.rb

```diff
@@ -15,11 +15,16 @@
 # along with Mapotempo. If not, see:
 # <http://www.gnu.org/licenses/agpl.html>
 #
+# frozen_string_literal: true
+
```

Comment:
Is this voluntary? We modify string variables so I don't know if freezing them can cause an issue. If there is no risk we need this for all files. Shall we do all now?

Reply:
no risk, should be added in all files, we have a code/task to add this un all file, but it's not available in optim-api